Monika After Story/game/script-windowreacts.rpy

```diff
@@ -562,3 +562,1927 @@ label mas_wrs_twitch:
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_aseprite:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Oooh, making some art? Nice! I bet it'll turn out great!",
+           "Making art simply using multicolored blocks sounds like a challenging task.\n You've got quite some talent, I must say!"    
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_aseprite')
+    return     
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_steam",
+            category=['steam'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_steam:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Looking for some new games to play, [player]?",
+           "Choo choo...Chu~ Ahaha, got you!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_steam')
+    return     
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_spotify",
+            category=['spotify'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_spotify:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Nice music selection, [player]! Mind if I listen along with you?",
+           "What are you listening to, [player]?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_spotify')
+    return     
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_itunes",
+            category=['itunes'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_itunes:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Nice music selection, [player]! Mind if I listen along with you?",
+           "What are you listening to, [player]?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_itunes')
+    return       
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_garry",
+            category=['garry'smod'],
```

Comment:
I thought we got rid of gmod.

Reply:
Whoopsie! I didn't catch that in the list. My apologies; I'll go remove it.